Source/Core/CorridorGeometry.js

```diff
@@ -622,6 +628,109 @@ define([
+        Cartesian3.normalize(direction, direction);
+        var normal = ellipsoid.geodeticSurfaceNormal(position1, scratchCartesian2);
+        var offsetDirection = Cartesian3.cross(direction, normal, scratchCartesian1);
+        Cartesian3.multiplyByScalar(offsetDirection, halfWidth, offsetDirection);
+
+        var minLat = min.latitude;
+        var minLon = min.longitude;
+        var maxLat = max.latitude;
+        var maxLon = max.longitude;
+
+        // Compute 2 offset points
+        Cartesian3.add(position1, offsetDirection, scratchCartesian2);
+        ellipsoid.cartesianToCartographic(scratchCartesian2, scratchCartographic);
+
+        var lat = scratchCartographic.latitude;
+        var lon = scratchCartographic.longitude;
+        minLat = Math.min(minLat, lat);
+        minLon = Math.min(minLon, lon);
+        maxLat = Math.max(maxLat, lat);
+        maxLon = Math.max(maxLon, lon);
+
+        Cartesian3.subtract(position1, offsetDirection, scratchCartesian2);
+        ellipsoid.cartesianToCartographic(scratchCartesian2, scratchCartographic);
+
+        lat = scratchCartographic.latitude;
+        lon = scratchCartographic.longitude;
+        minLat = Math.min(minLat, lat);
+        minLon = Math.min(minLon, lon);
+        maxLat = Math.max(maxLat, lat);
+        maxLon = Math.max(maxLon, lon);
+
+        min.latitude = minLat;
+        min.longitude = minLon;
+        max.latitude = maxLat;
+        max.longitude = maxLon;
+    }
+
+    var scratchCartesianOffset = new Cartesian3();
+    var scratchCartesianEnds = new Cartesian3();
+    var scratchCartographicMin = new Cartographic();
+    var scratchCartographicMax = new Cartographic();
+
+    function computeRectangle(positions, ellipsoid, width, cornerType) {
+        var length = positions.length - 1;
+        if (length === 0) {
+            return new Rectangle();
+        }
+        var halfWidth = width * 0.5;
+
+        scratchCartographicMin.latitude = Number.POSITIVE_INFINITY;
+        scratchCartographicMin.longitude = Number.POSITIVE_INFINITY;
+        scratchCartographicMax.latitude = Number.NEGATIVE_INFINITY;
+        scratchCartographicMax.longitude = Number.NEGATIVE_INFINITY;
+
+        if (cornerType === CornerType.ROUNDED) {
+            // Compute start cap
+            var first = positions[0];
+            Cartesian3.subtract(first, positions[1], scratchCartesianOffset);
+            Cartesian3.normalize(scratchCartesianOffset, scratchCartesianOffset);
+            Cartesian3.multiplyByScalar(scratchCartesianOffset, width, scratchCartesianOffset);
+            Cartesian3.add(first, scratchCartesianOffset, scratchCartesianEnds);
+            computeOffsetPoints(scratchCartesianEnds, first, ellipsoid, halfWidth,
+                scratchCartographicMin, scratchCartographicMax);
+        }
+
+        // Compute the rest
+        for (var i = 0; i < length; ++i) {
+            computeOffsetPoints(positions[i], positions[i+1], ellipsoid, halfWidth,
```

Comment:
@tfili was this addressed?  Do we need to do anything here?  Perhaps write an issue for a future PR if it is still a problem.